src/Common/TurbulenceClosures/TurbulenceClosures.jl

```diff
@@ -941,4 +1000,84 @@ function flux_second_order!(
     flux.ρe += hyperdiffusive.hyperdiffusion.ν∇³u_h * state.ρu
     flux.ρe += hyperdiffusive.hyperdiffusion.ν∇³h_tot * state.ρ
 end
+
+###
+### Divergence Damping Models
+###
+
+""" 
+    NoDivergenceDamping <: DivergenceDampingModel 
+
+Default do-nothing methods
+"""
+struct NoDivergenceDamping <: DivergenceDampingModel end
+
+"""
+    ConstantCoeffDivergenceDamping{FT} <: DivergenceDampingModel
+
+Computes fluxes for horizontal divergence damping term, 
+# Fields
+
+$(DocStringExtensions.FIELDS)
+
+"""
+struct ConstantCoeffDivergenceDamping{FT} <: DivergenceDampingModel
+    "Horizontal Divergence Damping Coefficient"
+    νd_h::FT
+    "Vertical Divergence Damping Coefficient"
+    νd_v::FT
+end
+vars_state(::ConstantCoeffDivergenceDamping, ::Gradient, FT) = @vars(ρu::SVector{3,FT})
+vars_state(::ConstantCoeffDivergenceDamping, ::GradientFlux, FT) = @vars(∇ρu::SMatrix{3,3,FT,9})
+
+function init_aux_divdamping!(
+    ::ConstantCoeffDivergenceDamping,
+    ::BalanceLaw,
+    aux::Vars,
+    geom::LocalGeometry,
+)
+    nothing
+end
+function compute_gradient_argument!(
+    m::ConstantCoeffDivergenceDamping,
+    bl::BalanceLaw,
+    transform::Vars,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+    transform.divergencedamping.ρu = state.ρu
+end
+
+function compute_gradient_flux!(
+    m::ConstantCoeffDivergenceDamping,
+    diffusive::Vars,
+    ∇transform::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+    diffusive.divergencedamping.∇ρu = ∇transform.divergencedamping.ρu
+end
+
+function flux_second_order!(
+    m::ConstantCoeffDivergenceDamping,
+    bl::BalanceLaw,
+    flux::Grad,
+    state::Vars,
+    diffusive::Vars,
+    aux::Vars,
+    t::Real
+)       
+    FT = eltype(state)
+    k̂ = vertical_unit_vector(bl.orientation, bl.param_set, aux)
+    div = tr(diffusive.divergencedamping.∇ρu)
+    
+    νdd_h = (SDiagonal(1, 1, 1) - k̂ * k̂') * SVector{3,FT}(m.νd_h, m.νd_h, m.νd_h)
+    νdd_v = dot(SVector{3,FT}(m.νd_v, m.νd_v, m.νd_v), k̂) * k̂
+    
```

Comment:
There are still a couple of things I do not understand here. Why is `νdd_v` computed as a dot product times a vector and not directly `νdd_v = m.νd_v .* k̂`? In the current formulation, if `k̂=(0, 1/sqrt(2), 1/sqrt(2))`, then ` νdd_v=sqrt(2)* m.νd_v* k̂`. This does not seem correct.